packages/components/src/placeholder/style.scss

```diff
@@ -212,7 +212,7 @@
 
 	// By painting the borders here, we enable them to be replaced by the Border control.
 	@include placeholder-style();
-	overflow: auto;
+	overflow: hidden;
```

Comment:
Overflow hidden seems risky here as it could cause some controls to be inaccessible via the UI 🤔

Would it make sense to only apply overflow hidden when the block is not selected/reset it to auto when the block is selected?

Reply:
They are already hidden in the UI, this is not accessible in any way shape or form:

![Screenshot 2024-03-14 at 09 48 22](https://github.com/WordPress/gutenberg/assets/1204802/c3261f10-b1ff-45c2-8d4d-2345773537d8)

This is why we've increasingly been careful about using the generic Placeholder pattern, and in fact why the Site Logo and Featured Image placeholders are patterns to follow for the best results, rather than the existing component. Those feature only a single button, acknowledging the fact that so long as you can insert a block into a colum that's 50x50px, they simply need to scale. There's a bit more discussion here: https://github.com/WordPress/gutenberg/pull/58685#issuecomment-1951006494

I hear you though, there may be a larger refactor necessary of the image block component. Perhaps #59275 can lead the way?

Alternatively I'd recommend a revert of the original PR to fix it.

Reply:
> I would love a refactor of this. But I think an immediate fix would be to only apply the overflow: hidden when the block is not selected. That would still prevent the layout shift but still retain the ability to scroll this area here

I have to temporarily move on to a separate thing, but good point, I can try this later. Otherwise if you're up for it feel free to push directly to the branch.

Reply:
Seems to work, what do you think?

![scrollbars only when selected](https://github.com/WordPress/gutenberg/assets/1204802/9aba72ed-e4d3-4876-92ca-91073a40c318)